src/transformers/models/vision_encoder_decoder/configuration_vision_encoder_decoder.py

```diff
@@ -15,8 +15,15 @@
 # limitations under the License.
 
 import copy
+from typing import Any, Mapping, Optional, OrderedDict
+
+from packaging import version
+
+from transformers.tokenization_utils_base import PreTrainedTokenizerBase
```

Comment:
We use relative imports in the `transformers` codebase, so please replace these imports. Also, for imports that are just used for type checking, we use:

```
if TYPE_CHECKING:
    from ... import PreTrainedTokenizerBase, TensorType
```

and then wrap the types in quotes ""

Reply:
Done